Gems/EMotionFX/Code/EMotionFX/Source/BlendTreeFabrikNode.h

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+// include the required headers
+#include "EMotionFXConfig.h"
+#include "AnimGraphNode.h"
+
+
+namespace EMotionFX
+{
+    /**
+     *
+     */
```

Comment:
This would not be a bad place to insert a description and where the code came from, if any, or a link to information about Fabrik.